site/docs/v1/tech/tutorials/integrate-expressjs.md

```diff
@@ -0,0 +1,153 @@
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='YOUR_API_KEY_FROM_ABOVE'
+npm run setup
+```
+
+If you want, you can [log into your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
+
+First, make a directory.
+
+```shell
+mkdir ../setup-expressjs && cd ../setup-expressjs
+```
+
+Create a `package.json` file with the following contents to set up the dependencies.
+
+```json
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/package.json %}
+```
+
+Now, change into the `setup-expressjs` directory and install the needed packages.
+
+```shell
+cd setup-expressjs && npm install
+```
+
+You are going to create some files in different directories, so pay attention to the final directory structure that you should have after completing these steps. 
+
+```
+├── app.js
+├── bin
+│   └── www
+├── package.json
+├── package-lock.json
+├── routes
+│   └── index.js
+└── views
+    ├── error.pug
+    ├── index.pug
+    └── layout.pug
+```
+
+To set up the web server, create a directory called `bin` and add the following to a `www` file.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/bin/www %}
+```
+
+To define pages for your application, create a `routes` folder and an `index.js` file there with the contents below.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/routes/index.js %}
+```
+
+The `views` directory will hold files responsible for rendering the pages.
+
+First, create a file called `layout.pug` that will contain the main structure for your application.
+
+```
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/views/layout.pug %}
+```
+
+Now, paste these contents in an `index.pug` file that will be rendered when visiting the main home page.
+
+```
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/views/index.pug %}
+```
+
+Finally, add an `error.pug` file to render error messages.
+
+```
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/views/error.pug %}
+```
+
+Stitch everything together by creating an `app.js` file that will be the main entrypoint for your application.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/app.js %}
+```
+
+Once you’ve created these files, start up the {{page.technology}} application using this command:
+
+```shell
+npm start
+```
```

Comment:
Can you please do the following:

* add an image of what the user will see when they first start up the app
* add a log out button (and make sure setup-express.js configures the logout url)
* double check this list: https://github.com/FusionAuth/fusionauth-site/blob/master/DocsDevREADME.md#webapps to make sure you have done all of that

Reply:
Done all that

_PS: we already had the logout button, so I just added the URL to the setup script_